src/client/common/experiments/service.ts

```diff
@@ -113,13 +118,15 @@ export class ExperimentService implements IExperimentService {
         // so we need to perform these checks and send the corresponding telemetry manually.
         switch (this.getOptInOptOutStatus(experiment)) {
             case 'optOut': {
+                await this.experimentationService.isCachedFlightEnabled(experiment);
```

Comment:
Regardless of whether user has opted in/out manually, we need to check with the experiment service.
This way if a user falls into an experiment, the telemetry will line up with experiments on server.

Reply:
@jakebailey @kimadeline /cc

Reply:
@rchiodo Please re-review.